master/include/comm/communicator.hpp

```diff
@@ -134,15 +160,18 @@ inline void Communicator::resStateCallback(const uint8_t *buf) {
     bool go_switch = (buf[0] >> 1) & 0x01;
     bool go_button = (buf[0] >> 2) & 0x01;
 
+    DEBUG_PRINT("Received message from RES");
+
     DEBUG_PRINT_VAR(emg_stop1);
     DEBUG_PRINT_VAR(emg_stop2);
     DEBUG_PRINT_VAR(go_switch);
     DEBUG_PRINT_VAR(go_button);
 
     if (go_button || go_switch)
         _systemData->r2dLogics.processGoSignal();
-    else if (emg_stop1 || emg_stop2)
+    else if (!emg_stop1 || !emg_stop2) {
```

Comment:
maybe I'm just being dumb, but can you explain this if, it's emergency if both or one are false?

Reply:
The emergency signals are zeros